README.md

```diff
@@ -6,10 +6,39 @@ This action enables you to fetch Doppler secrets for use in your GitHub Actions.
 
 ## Configuration
 
-This action requires a [Doppler Service Token](https://docs.doppler.com/docs/service-tokens) to provide read-only access to secrets for a specific Config within a [Project](https://docs.doppler.com/docs/create-project).
+The action can be configured in two ways:
+
+* Service Token (recommended)
+* Personal Token with Project and Config
+
+### Service Token
+
+A [Doppler Service Token](https://docs.doppler.com/docs/service-tokens) provides read-only access to single config and is recommended due to its limited access scope.
```

Comment:
typo/nit: "provides read-only access to single config" -> "provides read-only access to a single config"